doc/admin-api-cn.md

```diff
@@ -21,7 +21,13 @@
 目录
 ===
 
-* [Route](#route)
+- [目录](#%e7%9b%ae%e5%bd%95)
```

Comment:
Please use original string `#%e7%9b%ae%e5%bd%95`

Reply:
I'm also puzzled here. It may be due to the problem of the markdown plugin, the title is automatically updated every time my saving.